src/main/scala/extracells/container/ContainerFluidStorage.java

```diff
@@ -321,9 +354,17 @@ public void onListUpdate() {}
 
 	@Override
 	public void postChange(IBaseMonitor<IAEFluidStack> monitor, Iterable<IAEFluidStack> change, BaseActionSource actionSource) {
-		this.fluidStackList = ((IMEMonitor<IAEFluidStack>) monitor).getStorageList();
-		new PacketFluidStorage(this.player, this.fluidStackList).sendPacketToPlayer(this.player);
-		new PacketFluidStorage(this.player, this.hasWirelessTermHandler).sendPacketToPlayer(this.player);
+		if (Instant.now().compareTo(this.LastUpdateTime.plus(250, ChronoUnit.MILLIS)) > 0) {
```

Comment:
250 is a magic number, put this as a static final named constant, or ideally a config option for the mod.

Another big issue: if many updates happen quickly, but then no updates happen they will be invisible:
```
Time ->
0 1 2 3 4 5 ... 250 251 252
C c c c - - ... - - -
```
Only the first change C would be sent, the later ones would fail this If, but if no change happens afterwards they never get sent.